### ISSUE TITLE
Why is it only allowed me to use find, I just want to filter the array

### ISSUE THREAD

**Issue by yoyo837** [sindresorhus/eslint-plugin-unicorn#797]
`    "eslint-plugin-unicorn": "^21.0.0",`

```js
const orders = findedOrders.filter((od) => {
  if (od.platformId === l2Platform.id) {
    return true;
  }
  if (isTicket) {
    return true;
  }
  if (od.platformSubIds) {
    return od.platformSubIds.split(",").includes(l2Platform.id.toString());
  }
  return false;
});

```

![image](https://user-images.githubusercontent.com/6134547/87891560-fb52ae80-ca6c-11ea-9450-1e745a69ec22.png)

**Comment by fisker** [comment]
Did you only use first element of `orders` in your code?

**Comment by yoyo837** [comment]
Ooh, yes, I did.

**Comment by fisker** [comment]
Like this rule?

**Comment by yoyo837** [comment]
> Like this rule?

It's feels good!
